03week/ticTacToe.js

```diff
@@ -23,26 +25,112 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
+
+
 function horizontalWin() {
+  if (board.includes("X", [0, 0], [0, 1], [0, 2])) {
```

Comment:
using includes() is only checking if "X" or "O" is in the array at 1 spot. if it is for example in position 1 it will return true, it is not checking if every position includes "X". Look up every() method, that might be better for what you are trying to do.